src/class-bulk-task.php

```diff
@@ -523,21 +523,21 @@ public function run_wp_post_query( array $args, callable $callable ): void {
 		// All systems go.
 		while ( $this->min_id < $this->max_id ) {
 			// Build the query object, but don't run it without the object hash.
-			$query = new WP_Query();
+			$this->query = new WP_Query();
```

Comment:
The change to `$this->query` doesn't seem strictly related to the feature being added with the PR. I might consider splitting the bugfix into its own PR, so that the PR for the current feature is simpler. On the small chance that the bugfix leads to effects that need to be tracked down, it'll be easier to identify the applicable change and to see the reasoning behind it.

Reply:
Agree! It felt minor and a non-issue, but I understand the motivation.